### ISSUE TITLE
//replace ID:ID not recognizing integers greater that 15

### ISSUE THREAD

**Issue by Dweebington** [GTNewHorizons/GT-New-Horizons-Modpack#18411]
### Your GTNH Discord Username

Dweebington

### Your Pack Version

2.7.0

### Your Server

Private Server/SP

### Java Version

Java 8

### Type of Server

None

### Your Expectation

When running the command on blocks that contain multiple ID strings, the second number should be recognized and the block replacement should complete. 

Syntax: //replace ####:XX ####:XX 

Usage: 
1. Make selection with //wand
2. //replace 3544:27 3544:29
 
Outcome; 
replace Inverted Blue Lamp with Inverted Green Lamp

### The Reality

1-15 are recognized but anything higher is not and the command fails. 

Syntax: //replace ####:XX ####:XX 
Usage: Usage: //replace 3544:27 3544:29 fails to recognize integers above 15 otherwise the command fails
For parameter 'mask': Invalid data value '27'

### Your Proposal

Allow the //replace command to recognize integers above 15. As well as any other commands that require a syntax of ####:XX.

### Final Checklist

- [X] I have searched this issue tracker and there is nothing similar already. Posting on a closed issue saying the bug still exists will prompt us to investigate and reopen it once we confirm your report.
- [X] I can reproduce this problem consistently by follow the exact steps I described above, or this does not need reproducing, e.g. recipe loophole.
- [X] I have asked other people and they confirm they also have this problem by follow the exact steps I described above, or this does not need reproducing, e.g. recipe loophole.

**Comment by chochem** [comment]
that does not sound like GTNH but is probably world edit right?

**Comment by Dweebington** [comment]
Yes, correct, I have updated the title as a result. This was the ticket page I was provided via the discord
